R/constructors-Node.R

```diff
@@ -0,0 +1,157 @@
+#' @include class-Node.R
+#' Create a Node Id
+#' 
+#' Because typing `NodeId(id = 'foo')` is annoying, this function is provided
+#' to make things easier. Now you can do `NodeId('foo')`
+#' 
+#' @param value string a unique identifier for the node
+#' @export 
+NodeId <- function(value) {
+  new("NodeId", value = value)
+}
+
+
+#' Create a NodeIdList
+#' 
+#' @param object Object containing list of node ids
+#' @export 
+setGeneric("NodeIdList", function(object, uniqueOnly = c(TRUE, FALSE)) standardGeneric("NodeIdList")) 
```

Comment:
what's the use case for having `uniqueOnly = FALSE`?

Reply:
I thout the validation of `NodeIdList` will return an error if the ids are not unique anyhow.
Oh is this saying "hey do you want me to unique-ify the ids you sent me?"

Reply:
for my own benefit, how might the silent errors get produced in this case?